### PR TITLE
Feature/add timeout to put template raw

### PR DESCRIPTION
**Pull request by eran-gil**
Just today when trying to start a windows service, the service won't start because it was indefinitely waiting for ElasticSearch to respond to the PutTemplateRaw request, and then I had to remove all the appender references to allow for the service to run.

### CODE REVIEW

File: src/log4stash/ElasticSearchAppender.cs [hunk 1]
```diff
@@ -85,19 +87,35 @@ public override void ActivateOptions()
 
             if (Template != null && Template.IsValid)
             {
-                _client.PutTemplateRaw(Template.Name, File.ReadAllText(Template.FileName));
+                if (!Template.ForcePut)
```
Comment (urielha) [inline src/log4stash/ElasticSearchAppender.cs]: Just drop the `if` and leave ` _finishedPuttingTemplate = !Template.ForcePut;`

File: src/log4stash/ElasticSearchAppender.cs [hunk 2]
```diff
@@ -85,19 +87,35 @@ public override void ActivateOptions()
...
             }
 
             ElasticFilters.PrepareConfiguration(_client);
 
             RestartTimer();
         }
 
+        private void PutTemplateToElastic()
+        {
+            if (_client.PutTemplateRaw(Template.Name, File.ReadAllText(Template.FileName)))
```
Comment (urielha) [inline src/log4stash/ElasticSearchAppender.cs]: Just:
```C#

_finishedPuttingTemplate = _client.PutTemplateRaw(Template.Name, File.ReadAllText(Template.FileName));

```

**Review by urielha — APPROVED** [review]
Please take a look at my comments

**Comment by eran-gil** [comment]
Done :)

**Comment by urielha** [comment]
Hi
I don't see any need to verify the Template is putted (via Force and etc..)
Because the verification could failed again and we anyway do the index operation after it.

Furthermore, there is a chance that the **index operation** would fail too! so I think it is the user responsibility to check whether his template is inserted correctly and updated to his last version.

**Pull request merged**